sync_qubes-os.org_repo.sh

```diff
@@ -19,11 +29,13 @@ for rel in $RELS_TO_SYNC; do
         rsync_args="$rsync_args $rel/$repo"
     done
     echo "Syncing $rel..."
-    rsync $DRY --partial --progress --hard-links --exclude repodata -air $rsync_args $HOST:$HOST_BASEDIR/$rel/
-    rsync $DRY update_repo.sh update_repo-arg.sh mirrors.list $HOST:
-    for repo in $REPOS_TO_SYNC; do
-        [ -z "$DRY" ] && ssh $HOST sh -c "\"cd $HOST_BASEDIR; ~/update_repo-arg.sh $rel/$repo/dom0/fc* $rel/$repo/vm/*\""
-    done
+    if [ -z "$DRY" ]; then
+        for repo in $REPOS_TO_SYNC; do
+            "$dir/update_repo-arg.sh" "$rel/$repo/dom0"/fc* "$rel/$repo/vm"/*
+        done
+    fi
```

Comment:
I think you can skip this step now, local metadata is created in the other script already.